### DISCUSSION TITLE
Improving the design of the Recharts website/brand

### DISCUSSION THREAD

**Discussion by wesoudshoorn** [recharts/recharts#3787] [Ideas]
Hi everyone,

I'm Wes Oudshoorn, founder of AppSignal. I'm a designer/frontend developer by trade and co-founder of AppSignal. We're happily adopting Recharts as a library. I was wondering: Could do something back for the project?

I'm not a developer, but I can help out with the design/brand of Recharts. @nikolasrieble suggested I ask the community here your thoughts on Recharts's design/homepage/branding. Having a great library presentation can increase interest and adoption.

I'll share some of my thoughts on this, and I am curious what everyone thinks about this.

**The brand/logo**
A great-looking icon, logo, and brand can help Recharts get recognized. Just think about your favorite languages and frameworks; they all have something recognizable. The <Recharts/> logo has no such thing.

_Proposal:_ Create a new icon/logo.

**The homepage**
Coming to the homepage, folks should 
- Know what the library is
- What makes it unique
- Be wowed with some examples
- Be guided to a getting started guide

_Proposal:_ Redesign the homepage, with special attention to getting people excited about the library on the first page. Obviously, we keep our new Storybook setup.

**Examples**
Beautiful-looking charts are going to get more people to love Recharts. A library like [Tremor](https://www.tremor.so) does a great job of making Recharts look great. Some of the examples in Storybook can be nicer, and I believe that, especially on the homepage, we should show how great Recharts could look.

_Proposal:_ Design some great-looking examples for the homepage, and do a round of beautifying the examples in Storybook.
_Proposal:_ In the longterm, a gallery of ready-to-use, beautiful examples could be a great addition to the project. 

-----------

I would be happy to help out with this on the design part. In my experience, design is even more opinionated than code. There are ∞+1 ways to design something (while code may get stuck on ∞ 😉). I am looking forward to getting everyone's input and for the library maintainers to decide if they want to do something with this.

**Comment by nikolasrieble** [comment]
I think all three are of high value and would be great to have. 

## Logo 
Most straightforward to do. To my understanding probably an SVG icon, that would simply be used everywhere. As this is a bigger change to the way recharts is represented, I would appreciate the consent from long time maintainers, such as @arcthur. 

## Website 
Redoing the website has highest value, since it is the first point of contact for most people. It provides a first impression and has high influence on whether or not people choose recharts.
The website also seems like something that could be defined as multiple separate tasks. Merely scoping this out would already provide value. 

## Examples
Yes ❤️ 

Both for examples and website, I believe we can just start with iterative improvements. 

Curious to hear your perspective, @arcthur @ckifer

**Reply by ckifer** [reply]
Agreed on all fronts - I think we can improve a lot but still keep a similar "theme" if you will? I want to be respectful to the original creators (@arcthur) and what recharts was when it first came out. We also have a ton of issues like accessibility, responsivity, discoverability, etc. 

A logo is totally the easiest thing to do and maybe we could do some POC for a few variants and vote on them! We don't want it to be a corporate type thing, but something simple (highcharts logo as example even though they are a company)
![image](https://github.com/recharts/recharts/assets/25180830/17b53e8a-8c9a-4f1c-a74f-c73b7a786f2e)
Such a recognizable thing that looks cool in context!

The website is interesting because we have 2 things going right now 1. the OG website from scratch and 2. the storybook

IMO we either need to do our best to theme the storybook and move to that to recharts.org, or we need to use storybook as strictly a place for examples and move the website to something easier to manage like docusaurus. Not sure any other great paths forwards - totally redoing the current website is probably out of the question as more of a time and maintainability constraint than anything.
